packages/react-core/src/components/MultipleFileUpload/examples/MultipleFileUploadBasic.tsx

```diff
@@ -53,6 +54,17 @@ export const MultipleFileUploadBasic: React.FunctionComponent = () => {
     setReadFileData(newReadFiles);
   };
 
+  /** this function causes uploaded files to become corrupted if the corresponding option is selected in the example,
```

Comment:
```suggestion
  /** this function forces uploaded files to become corrupted if "Force uploads to fail" is selected in the example,
```

Reply:
Note - whatever the label ends up being, it would be good to update the comment here since it looks like this is the only scenario where we deliberately fail the upload.